src/script/media/MediaStreamHandler.js

```diff
@@ -807,21 +808,6 @@ export class MediaStreamHandler {
   // Media handling
   //##############################################################################
 
-  /**
-   * Check for active calls that need a MediaStream.
-   * @returns {boolean} Returns true if an active media stream is needed for at least one call
-   */
-  mediaStreamInUse() {
```

Comment:
This method also becomes obsolete because the mediaStream is not shared anymore. No need to check if it's "in use" by some other part of the code, only this class really "needs" it